acceptance/tests/ssl/autosign_command.rb

```diff
@@ -32,9 +32,22 @@ def assert_key_generated(name)
   fqdn = master.execute('facter fqdn')
 
   step "Step 1: ensure autosign command can approve CSRs" do
+
+    # Our script needs to consume stdin (SERVER-1094)
+    # We should revert back to /bin/true once resolved
+    autosign_true_script = <<-EOF
+!#/bin/bash
+less
```

Comment:
Yeah I was suspicious about this.  I like the explicit readlines too, I'll change to this.